testing/cfitsio/APKBUILD

```diff
@@ -0,0 +1,55 @@
+# Contributor: Holger Jaekel <holger.jaekel@gmx.de>
+# Maintainer: Holger Jaekel <holger.jaekel@gmx.de>
+pkgname=cfitsio
+pkgver=3.47
+pkgrel=0
+pkgdesc="A library of C and Fortran subroutines for reading and writing data files in FITS (Flexible Image Transport System) data format"
```

Comment:
Could you cut that description to ~80 chars please?

Reply:
cut to 87 characters